index.d.ts

```diff
@@ -0,0 +1,449 @@
+import React from "react";
+import Carousel from "react-native-snap-carousel";
+import dayjs from "dayjs";
+import RNPopover from "react-native-popover-view";
+import {
+  FlexboxProps,
+  SpaceProps,
+  BorderProps,
+  ColorProps,
+  LayoutProps,
+  PositionProps,
+  ButtonStyleProps,
+  TypographyProps,
+} from "styled-system";
+import {
+  ImageProps as RNImageProps,
+  ViewProps as RNViewProps,
+  TextInputProps as RNTextInputProps,
+  ScrollViewProps as RNScrollViewProps,
+  FlatListProps as RNFlatListProps,
+  TextProps as RNTextProps,
+  TextInputAndroidProps as RNTextInputAndroidProps,
+  TextInputIOSProps as RNTextInputIOSProps,
+  ViewStyle,
+  TextStyle,
+  TouchableOpacityProps,
+} from "react-native";
+import { ModalProps } from "react-native-modal";
+import { BottomTabBarProps } from "@react-navigation/bottom-tabs";
+import { CalendarProps as RNCalendarProps } from "react-native-calendars";
+import { FlashListProps as RNFlashListProps } from "@shopify/flash-list";
+import {
+  RichEditorProps,
+  RichToolbarProps,
+} from "react-native-pell-rich-editor";
+import { MaterialTopTabBarProps } from "@react-navigation/material-top-tabs";
+import {
+  ToastProps as RNToastProps,
+  ToastHideParams,
+  ToastShowParams,
+  ToastConfig,
+} from "react-native-toast-message";
+import { RippleProps } from "react-native-material-ripple";
+
+import { theme as themeDef } from "./src/theme";
+import { BUTTON_VARIANTS } from "./src/components/Button";
+
+type addPrefixToObject<
+  TObject extends object,
+  TPrefix extends string
+> = `${TPrefix}${keyof TObject}`;
+
+const theme: {
+  colors: {
+    font: addPrefixToObject<typeof themeDef.colors.font, "font.">;
+    background: addPrefixToObject<
+      typeof themeDef.colors.background,
+      "background."
+    >;
+    border: addPrefixToObject<typeof themeDef.colors.border, "border.">;
+    toast: addPrefixToObject<typeof themeDef.colors.toast, "toast.">;
+  };
+  fonts: keyof typeof themeDef.fonts;
+  fontSizes: keyof typeof themeDef.fontSizes;
+};
+
+interface StyleProps
+  extends FlexboxProps,
+    SpaceProps,
+    BorderProps,
+    ColorProps,
+    LayoutProps,
+    PositionProps {}
+
+interface ViewProps extends RNViewProps, StyleProps {
+  children?: React.ReactNode;
+}
+interface TextProps extends RNTextProps, StyleProps, TypographyProps {
+  children?: React.ReactNode;
+}
+interface TextInputProps
+  extends RNTextInputAndroidProps,
+    RNTextInputIOSProps,
+    StyleProps,
+    TypographyProps {}
+interface FlatListProps extends RNFlatListProps, StyleProps {}
+interface ScrollViewProps extends RNScrollViewProps, StyleProps {}
+interface TouchableProps extends RippleProps, StyleProps, ButtonStyleProps {
+  children?: React.ReactNode;
+}
+
+interface AccordianProps extends ViewProps {
+  header?: () => void;
```

Comment:
the header is component, right?
```suggestion
  header?: React.ReactNode;
```

Reply:
@sangameshsomawar header actually accepts a function.
https://github.com/bigbinary/neeto-ui-rn/blob/1856bc512ab06aaaceab03ab2b640db2005eee02/src/components/Accordion.js#L152